### ISSUE TITLE
Rod Aimbot

### ISSUE THREAD

**Issue by rxversedcode** [ImpactDevelopment/ImpactIssues#2741]
**Is your feature request related to a problem? Please describe.**
No, it is not.

**Describe the solution you'd like**
I would like a "Rod Aimbot" hack under the "Combat" section. I would like that because rods are used in PvP a lot. For example, BuildUHC.

**Describe alternatives you've considered**
I'm pretty sure in vape client there is a rod aimbot hack.

**Additional context**
Nothing else.

**Comment by l1ving** [comment]
I don't think this is very useful but I'll keep it open anyways, Unpack is a module lol

**Comment by ZWORX52** [comment]
how would you tell when to aim? whenever the rod is held does not seem like the best idea as it could be bothering. and you might  forget that it is on and it would be bothering, idea what I'm saying lol

**Comment by Salihius2** [comment]
Rod aimbot is a pretty good idea for pvp.

**Comment by rxversedcode** [comment]
> I don't think this is very useful but I'll keep it open anyways, Unpack is a module lol



> how would you tell when to aim? whenever the rod is held does not seem like the best idea as it could be bothering. and you might forget that it is on and it would be bothering, idea what I'm saying lol

when you have the rod in your hand, it aims for the torso.

**Comment by l1ving** [comment]
that's not a when, that's a where 😐

**Comment by rxversedcode** [comment]
i am waiting

**Comment by bilawalali2111-svg** [comment]
this can be very useful
